### ISSUE TITLE
Unnecessary calls to _loadPixels() and _applyEffects() in Sprites

### ISSUE THREAD

**Issue by eonarheim** [excaliburjs/Excalibur#481]
This would make sprites way more efficient on draw. `_loadPixels()` and `_applyEffects` seem to be called even when they are completely loaded and effects have been applied. This can be seen in the console network tab as the base64 images are reloaded into the in memory image tag src attributes.

``` javascript
// we should avoid setting src as much as possible
 this.internalImage.src = this._spriteCanvas.toDataURL('image/png');
```

![image](https://cloud.githubusercontent.com/assets/612071/8537900/92c047c6-242a-11e5-8e97-3fda8d1db386.png)

![image](https://cloud.githubusercontent.com/assets/612071/8537920/cc169106-242a-11e5-9dc0-571a0aa9e456.png)

![image](https://cloud.githubusercontent.com/assets/612071/8537927/eba74844-242a-11e5-971e-fceee8117cd5.png)

**Comment by eonarheim** [comment]
We should measure the impact of this issue, perhaps with a visual test or a timing test in the chrome profiling tools

**Comment by eonarheim** [comment]
Closed by #1730
